website/src/pages/pricing.tsx

```diff
@@ -501,39 +425,42 @@ export default class Pricing extends React.Component<any, any> {
                                         </div>
                                     </div>
                                     <div className="row feature-table-row">
-                                        <div className="col-6 feature-title">Managed deployment option</div>
-                                        <div className="col-2">
-                                            <div className="table-blank" />
+                                        <div className="col-8 feature-title">
+                                            Managed deployment option (
+                                            <a href="/contact" target="_blank">
```

Comment:
Should this link to the trail page where customers can email us directly instead of our contact page? (https://about.sourcegraph.com/contact/sales/)